src/dev/code_coverage/ingest_coverage/process.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { fromEventPattern, of, fromEvent } from 'rxjs';
+import { concatMap, delay, map, takeUntil } from 'rxjs/operators';
+import jsonStream from './json_stream';
+import { pipe, noop, green, always } from './utils';
+import { ingest } from './ingest';
+import {
+  staticSite,
+  statsAndstaticSiteUrl,
+  addJsonSummaryPath,
+  testRunner,
+  addTimeStamp,
+  buildId,
+  coveredFilePath,
+  ciRunUrl,
+  itemizeVcs,
+} from './transforms';
+import { resolve } from 'path';
+import { createReadStream } from 'fs';
+import readline from 'readline';
+
+const back4Directories = '../../../..';
```

Comment:
I think going with something like `ROOT` as you did elsewhere is much easier to understand.

Reply:
Sure!